tests/functional/src/main/java/org/mule/tck/junit4/FunctionalTestCase.java

```diff
@@ -155,13 +155,13 @@ else if (flowConstruct instanceof AbstractPipeline)
         }
 
         throw new RegistrationException(
-            MessageFactory.createStaticMessage("Can't get component from flow construct "
-                                               + flowConstruct.getName()));
+                MessageFactory.createStaticMessage("Can't get component from flow construct "
```

Comment:
This class seems to have only formatting changes unrelated to this PR. There are other files like this one. Please remove them all from this PR